test/functional/migrations/generate-command/entity/post.entity.ts

```diff
@@ -23,4 +23,6 @@ export class Post extends BaseEntity {
     @JoinTable()
     categories: Category[];
 
+    @Column("decimal", { default: -0, precision: 3, scale: 1 })
```

Comment:
Is it possible to create another entry where there are multiple test cases?

A few that I can think of:

* Omitting the `scale`
* A bigger `default`
* A negative `scale`?  Is that possible?
* A negative default
* A string default
* A column of type int with a number default

Reply:
Did you mean it is necessary to add the tests for `default` feature thoroughly?
My objective is resolving those issues. I can't sure your suggestion corresponds to the pull request title.

Reply:
Oh - `string` default won't be necessary actually, you're right - since it won't hit this case.  But the others are still relevant - the issue I'm thinking of is that your change may affect other cases of when the default is a number and we did not expect.

The test case you have here only tests for one very specific scenario - however, other permutations exist that we should check for.

EG:

```
@Column("decimal", { default: -0 })
```

What happens when the scale is omitted and undefined?  We don't really know & the change you made may or may not handle it & we should add test cases to verify that functionality.

Reply:
Yeah, actually you suggestion is great. I'll add it.
I'm thinking about adding column to the new entity of the same test. Because I don't want to increase test time. Is it permitted?